erpnext/loan_management/doctype/loan_application/loan_application.py

```diff
@@ -16,6 +16,9 @@
 
 class LoanApplication(Document):
 	def validate(self):
+		if self.applicant_type == 'Employee':
+			from erpnext.loan_management.doctype.loan.loan import validate_employe_currency_with_company_currency
+			validate_employe_currency_with_company_currency(self.applicant, self.company)
```

Comment:
This validation is not required here. Because in loan application we don't know whether it will be repaid from salary.